server/odc-service/src/test/java/com/oceanbase/odc/service/common/WebRequestUtilsTest.java

```diff
@@ -49,4 +52,28 @@ public void isRedirectUrlValid_HostNotMatch_InValid() {
 
         Assert.assertFalse(redirectUrlValid);
     }
+
+    @Test
+    public void testGetTrulyClientIp() {
```

Comment:
test case's name suggestion: ${test method name}_${test scenarios}_${expected result}

this test case may be named: getTrulyClientIp_normalInput_getSucceed

Reply:
Would it be appropriate to name this test method, which encompasses both correct and erroneous test scenarios, as getTrulyClientIp_allTestCases?